unity/CAVE/Assets/W3D_Translator/Classes.cs

```diff
@@ -8,11 +11,13 @@
 
 namespace W3D
 {
+
     /********** STORY            ***********/
 
 
+    [Serializable]
     [XmlRoot(ElementName="Story")]
-    public class Story
+    public class Story : W3D
```

Comment:
What does this `: W3D` do?

Reply:
Ahh `W3D` is a base class. I'd consider moving the base class to the top of this file before the rest of the classes that inherit from this. I don't know C# or C++ style guides, but is it common to name a class the same name as the namespace? That caused me a little confusion

Reply:
Okay! I moved it and renamed it as `Xml`